tailwind.js

```diff
@@ -54,7 +54,8 @@ let colors = {
   maize: "#ffc450",
   purple: "#db3eb1",
   red: "#f93822",
-  white: "#ffffff"
+  white: "#ffffff",
```

Comment:
this file shouldn't be re-prettifyied.
I opened a PR which excludes this file from prettier formatting in https://github.com/garySiladi/chat-project/pull/18/files which will fix this
at the moment try to reset this file, momentarily disable the autoformat prettier extension, add your changes and commit